### ISSUE TITLE
Slider value out of range

### ISSUE THREAD

**Issue by NINI1988** [telerik/kendo-angular#1549]
## I'm submitting a...

* Bug report 

## Current behavior

When the slider is moved left or right of the component, the value gets sometimes out of the range.

## Expected behavior

The value of the slider should never be out of min max range.

## Minimal reproduction of the problem with instructions
https://plnkr.co/edit/RlXTv4dQAYCtplBS7W1j?p=preview
Move the slider left of the slider component, like in the picture below. The slider value gets sometimes a negative value.
![kendoslidererrorrange](https://user-images.githubusercontent.com/2767665/40407266-27f67bfa-5e64-11e8-810d-679b76af4971.PNG)

## What is the motivation or use case for changing the behavior?

It is a Bug and causes errors when using the slider value.
## Environment

Package versions:

```
    "@angular/animations": "^5.2.0",
    "@angular/common": "^5.2.0",
    "@angular/compiler": "^5.2.0",
    "@angular/core": "^5.2.0",
    "@angular/forms": "^5.2.0",
    "@angular/http": "^5.2.0",
    "@angular/platform-browser": "^5.2.0",
    "@angular/platform-browser-dynamic": "^5.2.0",
    "@angular/router": "^5.2.0",
    "@progress/kendo-angular-buttons": "^4.0.0",
    "@progress/kendo-angular-dropdowns": "^2.1.0",
    "@progress/kendo-angular-inputs": "^2.2.0",
    "@progress/kendo-angular-intl": "^1.4.1",
    "@progress/kendo-angular-l10n": "^1.2.0",
    "@progress/kendo-drawing": "^1.5.1",
    "@progress/kendo-theme-bootstrap": "^2.11.10",
```

Browser:

- Chrome (desktop) version 66.0.3359.181 (Offizieller Build) (64-Bit)

System:
- TypeScript version: 1.0.3.0
- Node version: v8.11.2 , 5.8.0
- Platform:  Windows

**Comment by ipeshev** [comment]
Fixed in v3.1.3-dev.201806011000

**Comment by tsvetomir** [comment]
Fix released in v3.1.3
